WordPress/Classes/ViewRelated/Post/Revisions/Browser/RevisionBrowserState.swift

```diff
@@ -0,0 +1,26 @@
+class RevisionBrowserState {
```

Comment:
I'd suggest to document what this class represents, and what it's scope is (what does it take care of?).